From 7ec3262a3033c2c7f56118d9d7dab5d99f81bc27 Mon Sep 17 00:00:00 2001
From: Yazan Shhady <yazan.shhady@solid-run.com>
Date: Wed, 6 Jul 2022 17:27:02 +0300
Subject: [PATCH] Update mx6cuboxi uboot env - add fdtfile support for HB-CBI

---
 include/configs/mx6cuboxi.h | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/include/configs/mx6cuboxi.h b/include/configs/mx6cuboxi.h
index 5e1fe5e760..e8f1782e16 100644
--- a/include/configs/mx6cuboxi.h
+++ b/include/configs/mx6cuboxi.h
@@ -101,6 +101,7 @@
 	"initrd_high=0xffffffff\0" \
 	"fdt_high=0xffffffff\0" \
 	"ip_dyn=yes\0" \
+	"hb_cbi=no\0" \
 	"console=" CONSOLE_DEV ",115200\0" \
 	"bootm_size=0x10000000\0" \
 	"mmcdev=" __stringify(CONFIG_SYS_MMC_ENV_DEV) "\0" \
@@ -125,10 +126,12 @@
 			"setenv fdtprefix imx6dl; fi; " \
 		"if test ${som_rev} = V15; then " \
 			"setenv fdtsuffix -som-v15; fi; " \
+		"if test ${hb_cbi} = yes; then " \
+			"setenv boardsuffix -cbi; fi; " \
 		"if test ${has_emmc} = yes; then " \
 			"setenv emmcsuffix -emmc; fi; " \
 		"if test ${board_name} = HUMMINGBOARD2 ; then " \
-			"setenv fdtfile ${fdtprefix}-hummingboard2${emmcsuffix}${fdtsuffix}.dtb; fi; " \
+			"setenv fdtfile ${fdtprefix}-hummingboard2${boardsuffix}${emmcsuffix}${fdtsuffix}.dtb; fi; " \
 		"if test ${board_name} = HUMMINGBOARD ; then " \
 			"setenv fdtfile ${fdtprefix}-hummingboard${emmcsuffix}${fdtsuffix}.dtb; fi; " \
 		"if test ${board_name} = CUBOXI ; then " \
-- 
2.25.1

